app/channels/test/solutions_channel.rb

```diff
@@ -0,0 +1,21 @@
+module Test
```

Comment:
Scoped under test as we'll need to create an actual `SolutionsChannel` which does everything correctly. This test channel will just return stubbed responses.